picard/metadata.py

```diff
@@ -135,10 +134,11 @@ def compare_to_release_parts(self, release, weights):
             except ValueError:
                 pass
             else:
-                if "title" in weights:
-                    b = int(release.medium_list[0].medium[0].track_list[0].count)
-                else:
-                    b = int(release.medium_list[0].track_count[0].text)
+                if 'media' in release:
```

Comment:
Is `media` ever not present here? Given that `totaltracks` is known at this point, this does look weird.